src/neutron/libGudrunN/groups_routines.f90

```diff
@@ -37,6 +37,7 @@ MODULE groups_routines
     integer, dimension(2)                   :: olddims,newdims
     integer                                 :: oldsize,newsize
     integer, dimension(:), allocatable      :: igroup_in
+    integer                                 :: start !start position to extract calibration filename from
```

Comment:
That's a very generic name! Perhaps calling it `fname_start` would be cleaner?

Reply:
Of course!